curvefs/src/metaserver/s3compact_wq_impl.cpp

```diff
@@ -120,14 +124,15 @@ S3CompactWorkQueueImpl::BuildValidList(const S3ChunkInfoList& s3chunkinfolist,
                                        uint64_t inodeLen) {
     std::list<struct S3CompactWorkQueueImpl::Node> validList;
     std::function<void(uint64_t, uint64_t, uint64_t, uint64_t, uint64_t,
-                       uint64_t)>
+                       uint64_t, uint64_t)>
```

Comment:
does these seven parameters are corresponding to fields in `S3CompactWorkQueueImpl::Node`, if so, why not use Node directly?

Reply:
good idea. fixed.